installer/omnisharp-manager.ps1

```diff
@@ -11,7 +11,7 @@
 [CmdletBinding()]
 Param(
     [Parameter()][Alias('v')][string]$version,
-    [Parameter()][Alias('l')][string]$location = "C:\omnisharp\",
+    [Parameter()][Alias('l')][string]$location = "$($env:USERPROFILE)\.omnisharp\omnisharp-roslyn\",
```

Comment:
This line only sets the default install location of the server, OmniSharp-vim gives the location it wants to install the server by using the `-l` option.

I designed the script to be as generic as possible, so it would be great if you could remove the `\omnisharp-roslyn` from the default install path.

Reply:
I can revert this back if you think it is best defaulting to C:\. I made the change so that running the script directly (with no arguments) vs when omnisharp-vim called it you would get the same result. Then when omnisharp-vim checked for it on opening a .cs file it wouldn't ask to install a second instance. But I can include more detail in the README.md about what the script is doing and to include the path as an argument.

Reply:
@atomsand1101its, I should have made that comment a bit clearer. The `$($env:USERPROFILE)` is a great addition, I just have a problem with the `\omnisharp-roslyn` on the end of the path.

So this would be fine:

```powershell
[Parameter()][Alias('l')][string]$location = "$($env:USERPROFILE)\.omnisharp\",
```

I think it might be best to explain in more depth in the `README,md` file about using the `-l` and the `-H` options (`-H` installs the HTTP version of the server which OmniSharp-vim needs).

However once the user has allowed the execution of PowerShell scripts, they should just be able to use `:OmniSharpInstall` to run the script with the correct parameters.

Reply:
I changed around the readme file to be more generic and included the usage description for the powershell script. It includes the process and gives the suggestion to let omnisharp-vim to do the install. I didn't mention the use of the `:OmniSharpInstall` because I had to be in .cs file to run that command and by going to the .cs file it automatically offers the install. I also have it defaulting to the .\omnisharp\ directory.